tests/virt_autotest/update_package.pm

```diff
@@ -71,6 +71,9 @@ sub run {
         save_screenshot;
     }
 
+    script_run "sed -i 's/memsize=2048/memsize=1024/g' /usr/share/qa/virtautolib/lib/virt-install.sh";
```

Comment:
Why needing this? We recently merged julie's PR which changes default mem size to 2048 on purpose. Why changing it back to 1024 here?

Reply:
this code changed just help to trigger the known guest installation issue to validate this PR as the finial validation run. 

For now, had finished the following final validation run for this PR:
[gi-guest_sles15sp2-on-host_developing-kvm](http://149.44.176.58/tests/5687497)
[gi-guest_sles15sp2-on-host_developing-xen](http://149.44.176.58/tests/5687500) 

Also, recover to Julie's PR used with the default guest mem size to 2048 on purpose now,